SQL/CreateAllTables.sql

```diff
@@ -30,17 +31,16 @@ CREATE TABLE plan_contains
 
 CREATE TABLE plan
 (
-  plan_id     INT NOT NULL AUTO_INCREMENT,
+  plan_id     INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
```

Comment:
Why did you change the PRIMARY KEY definition like this? The rest of the codebase declared it like PRIMARY KEY (plan_id)